services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/api_client/_thin.py

```diff
@@ -222,3 +222,11 @@ async def post_containers_tasks_restart(
     ) -> Response:
         url = self._get_url(dynamic_sidecar_endpoint, "/containers:restart")
         return await self.client.post(url)
+
+    @retry_on_errors
+    @expect_status(status.HTTP_204_NO_CONTENT)
+    async def post_docker_quotas_supported(
```

Comment:
I think that for custom operations (i.e. those that doe not correspond to CRUD) it is better to name the functions in the ``_thin`` interface with the operation itself.

For instance imaging a resource ``/foo``. The thin interface would be
```python
# CRUD operations
async def list_foo(...):
  # GET /foo

async def get_foo(...): 
    # GET /foo/*

async def create_foo(...): 
    # POST /foo/*

async def update_foo(): 
    # PUT /foo/*

async def delete_foo(...): 
    # DELETE /foo/*

# CUSTOM OPERATORS
async def check_foo(): 
    # PATCH /foo/*:check

```